material-ui/example/src/examples/nested/form-data.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Also, do we need the examples directory? Could we potentially merge this with the playground?

Reply:
Yes, of course. This isn't necessary when we have separate the core between themes.